jsesc.js

```diff
@@ -71,7 +71,7 @@ const singleEscapes = {
 const regexSingleEscape = /["'\\\b\f\n\r\t]/;
 
 const regexDigit = /[0-9]/;
-const regexWhitelist = /[ !#-&\(-\[\]-~]/;
+const regexWhitelist = /[ !#-&\(-\[\]-_a-~]/;
```

Comment:
This file is generated and should not be modified directly.

Instead, could you please update https://github.com/mathiasbynens/jsesc/blob/master/src/data.js accordingly, and run the build script?